specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/examples/amlComputeListNodes.json

```diff
@@ -4,7 +4,7 @@
     "resourceGroupName": "testrg123",
     "workspaceName": "workspaces123",
     "computeName": "compute123",
-    "api-version": "2018-11-19"
+    "api-version": "2019-05-01"
```

Comment:
This is WIP, but I think it would need to be moved in a new version folder to change this